docs/guides/websites/hosting/host-a-website-with-high-availability/index.md

```diff
@@ -20,27 +20,31 @@ image: host-a-website-with-high-availability-title-graphic.jpg
 authors: ["Phil Zona"]
 ---
 
+{{< note type="warning" >}}
+This guide is designed to be used in data centers that support the legacy ARP-based failover method. For a list of data centers supporting this method, see [IP Sharing Availability](/docs/products/compute/compute-instances/guides/failover/#ip-sharing-availability). If you are located in a data center that uses the newer BGP-based failover method, the steps in the Keepalived section of this guide do not apply. Instead, review the [Configure Failover on a Compute Instance](/docs/products/compute/compute-instances/guides/failover/#configure-failover) for IP Sharing and failover instructions.
```

Comment:
I'm not entirely sure here what a customer in a BGP DC needs to do. Is the instruction to follow all of these rules until the Keepalived section, then jump to the product docs?

Reply:
Ah. This notice wasn't intended to provide feature-complete instructions, just to notify readers that this guide is not designed for BGP DCs and to give them an additional resource to use. I do agree that this could be clearer, so I reworked this notice and added another notice in the Keepalived section.